spark/common/pom.xml

```diff
@@ -95,6 +95,11 @@
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.xerial</groupId>
+            <artifactId>sqlite-jdbc</artifactId>
+            <version>3.36.0.3</version>
```

Comment:
Just a heads-up: this jar is 13MB+ in terms of size.

Reply:
@jiayuasu do you think we need to change scope to provided and the user decides if it should be included. In that case I need to adjust the documentation.

Reply:
Let's still include it as we don't want to introduce more friction to our users